### PR TITLE
Beihang homepage is now under https

### LINKED ISSUE #797: Bug in rule “headers.copyright”: Missing link 'http://ev.buaa.edu.cn/' with text 'Beihang'.
http://ev.buaa.edu.cn/ has an HTTPS version now. Could you please update the rule to match that URL? Thanks!

Found [while checking `https://drafts.csswg.org/css-2018/`](https://www.w3.org/pubrules/?url=https%3A%2F%2Fdrafts.csswg.org%2Fcss-2018%2F&profile=WG-NOTE&validation=simple-validation&noRecTrack=true&informativeOnly=true&echidnaReady=false&patentPolicy=pp2004).

### PR DESCRIPTION
**Pull request by deniak**
Fix #797

### CODE REVIEW

**Comment by deniak** [comment]
I've submitted a [PR for respec](https://github.com/w3c/respec/pull/2090) and I believe @xfq is taking care of bikeshed.
/cc @plehegar

File: lib/rules/headers/copyright.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ exports.check = function (sr, done) {
         ,   "http://www.csail.mit.edu/":                                        "MIT"
         ,   "http://www.ercim.eu/":                                             "ERCIM"
         ,   "http://www.keio.ac.jp/":                                           "Keio"
-        ,   "http://ev.buaa.edu.cn/":                                           "Beihang"
+        ,   "https://ev.buaa.edu.cn/":                                           "Beihang"
```
Comment (xfq) [inline lib/rules/headers/copyright.js]: nitpick: please align the word "Beihang" with the others

Comment (xfq) [reply lib/rules/headers/copyright.js]: It seems the Beihang website is the only https website here?

Comment (deniak) [reply lib/rules/headers/copyright.js]: @xfq it should now be fixed

**Comment by deniak** [comment]
@xfq ok to merge?

**Comment by xfq** [comment]
Merged.

**Comment by xfq** [comment]
Thank you!

**Pull request merged**